crates/polars-plan/src/dsl/builder_dsl.rs

```diff
@@ -231,6 +208,16 @@ impl DslBuilder {
         .into()
     }
 
+    /// Remove rows matching a filter predicate (note that rows
+    /// where the predicate resolves to `null` are *not* removed).
+    pub fn remove(self, predicate: Expr) -> Self {
```

Comment:
We don't use this anywhere right? I think we should remove it in that case.

Reply:
It's used in `drop_nans` to make the check correct/clean, above ;)